packages/static/src/tabs/tab.stories.tsx

```diff
@@ -0,0 +1,264 @@
+          <Tabs.Paneless
+            variant='simple'
+            items={items}
+            selectedKey={value}
+            onSelectionChange={(v: React.Key) => setValue(v)}
+          />
+          {value === '1' && <FetchList id='1' />}
+          {value === '2' && <FetchList id='2' />}
+          {value === '3' && <FetchList id='3' />}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export const H5: React.FC = () => {
+  const [value, setValue] = React.useState<React.Key>('1')
+  const items = [
+    { key: '1', title: 'Founding of Rome' },
+    { key: '2', title: 'Monarchy and Republic' },
+    { key: '3', title: 'Empire' },
+  ]
+  return (
+    <div style={{ ...style }}>
+      <h3>H5</h3>
+      <div style={{ flexDirection: 'row', ...style }}>
+        <div style={{ width: '100%', display: 'flex', flexDirection: 'column' }}>
+          <Tabs.Paneless
+            variant='h5'
+            items={items}
+            selectedKey={value}
+            onSelectionChange={(v: React.Key) => setValue(v)}
+          />
+          {value === '1' && <FetchList id='1' />}
+          {value === '2' && <FetchList id='2' />}
+          {value === '3' && <FetchList id='3' />}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export const Icon: React.FC = () => (
+  <div style={{ ...style }}>
+    <h3>With left icon</h3>
+    <div style={{ flexDirection: 'row', ...style }}>
+      <Tabs icon='left'>
+        <Tabs.Item key='1' title='Founding of Rome'>
+          Arma virumque cano, Troiae qui primus ab oris.
+        </Tabs.Item>
+        <Tabs.Item key='2' title='Monarchy and Republic'>
+          Senatus Populusque Romanus.
+        </Tabs.Item>
+        <Tabs.Item key='3' title='Empire'>
+          Alea jacta est.
+        </Tabs.Item>
+      </Tabs>
+    </div>
+
+    <h3>With right icon</h3>
+    <div style={{ flexDirection: 'row', ...style }}>
+      <Tabs icon='right' disabledKeys={['1']}>
+        <Tabs.Item key='1' title='Founding of Rome'>
+          Arma virumque cano, Troiae qui primus ab oris.
+        </Tabs.Item>
+        <Tabs.Item key='2' title='Monarchy and Republic'>
+          Senatus Populusque Romanus.
+        </Tabs.Item>
+        <Tabs.Item key='3' title='Empire'>
+          Alea jacta est.
+        </Tabs.Item>
+      </Tabs>
+    </div>
+
+    <h3>With top icon</h3>
+    <div style={{ flexDirection: 'row', ...style }}>
+      <Tabs variant='simple' icon='top' disabledKeys={['1']}>
+        <Tabs.Item key='1' title='Founding of Rome'>
+          Arma virumque cano, Troiae qui primus ab oris.
+        </Tabs.Item>
+        <Tabs.Item key='2' title='Monarchy and Republic'>
+          Senatus Populusque Romanus.
+        </Tabs.Item>
+        <Tabs.Item key='3' title='Empire'>
+          Alea jacta est.
+        </Tabs.Item>
+      </Tabs>
+    </div>
+  </div>
+)
+
+export const CustomCssModule: React.FC = () => (
+  <div style={{ ...style }}>
+    <h3>User can override css by create a replacement css module</h3>
```

Comment:
Paraphrase this to:
Users can customize the styling of the component in by applying their own CSS